xtask/src/querycheck.rs

```diff
@@ -14,17 +14,16 @@ pub fn query_check() -> Result<(), DynError> {
     ];
 
     for language in lang_config().language {
-        let language_name = language.language_id.to_ascii_lowercase();
         let grammar_name = language.grammar.unwrap_or(language.language_id);
         for query_file in query_files {
             let language = get_language(&grammar_name);
-            let query_text = read_query(&language_name, query_file);
+            let query_text = read_query(&language.language_id, query_file);
```

Comment:
`language` gets shadowed above which is causing the compilation to fail

Reply:
Thanks, sorry I missed that :)